.pre-commit-config.yaml

```
@@ -40,7 +40,7 @@ repos:
     #- id: cmake-lint <- not ready yet
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.0.282
+    rev: v0.0.284
```

```suggestion
    rev: v0.0.285
```
It looks like the second push by the bot tried to go up again, and you've undone their hard work.